README.md

```diff
@@ -1,6 +1,6 @@
 # Datura
 
-Welcome to this temporary documentation for Datura, a gem dedicated to transforming and posting data sources from CDRH projects.  This gem is intended to be used with a "data repository" containing TEI, VRA, CSVs, and more.
+Welcome to this temporary documentation for Datura, a gem dedicated to transforming and posting data sources from CDRH projects.  This gem is intended to be used with a collection containing TEI, VRA, CSVs, and more.
```

Comment:
:+1: I hadn't caught all those other places with "project" etc. I hope this makes things clearer for readers going forward :)